app/views/users/show.html.slim

```diff
@@ -65,9 +65,13 @@ nav.actions
     p = @user.company_info
 
   - if can_see_private_info?
-    - fields = ['tshirt_size', 'tshirt_cut', 'postal_address']
     .well.private-info
       h3 Private info
+      - if @user.postal_address
+        h4 Postal Address
+        p
```

Comment:
Has been some time that I read `slim` template code 🙈 

What's the point of this empty paragraph?  Should the postal-address partial rendered in a paragraph? If yes, then I think™ the way to go would be_
```ruby
p
  = render ...
```

But looking at the partial not using any markup at all, I'd rather suggest to do the markup in there.